### DISCUSSION TITLE
Noob question! Plugins disappeared in the web interface

### DISCUSSION THREAD

**Discussion by Kuton-bit** [netbox-community/netbox#16622] [Help Wanted!]
I updated netbox and installed plugins. The plugin menu disappeared in the web interface. 
And I can`t find plugins in web interface. But url-link to plugin menu work [example: acl-list] correct.

NetBox version: 4.0.5
![nb1](https://github.com/netbox-community/netbox/assets/173037034/72b59a34-2437-4b2e-b46e-4a2331c9c0c2)

![image](https://github.com/netbox-community/netbox/assets/173037034/ee264703-7b86-4c67-9552-28044fa8bbb6)

![image](https://github.com/netbox-community/netbox/assets/173037034/8b57cc2b-f35c-406d-995c-ba508f7dc38b)

**Comment by jonasnieberle** [comment]
From the netbox 3.x.x version to 4.x.x are some changes in the plugin development. Maybe you install some plugins which are not updated to the new netbox version ...
